src/main/java/com/softserveinc/ita/pageobjects/admin/ProtocolPage.java

```diff
@@ -46,29 +46,30 @@ public boolean isSearchButtonEnabled() {
         return searchButton.isEnabled();
     }
 
-    @Step("Protocol page: moving date picker backward")
-    public ProtocolPage moveDatePickerBackward(int index) {
+    private ProtocolPage moveDatePicker(int index, String direction) {
```

Comment:
placing variables and methods in order by access level:

Static Variables
Public Variables
Protected Variables
Private Variables
Constructors
Static Methods
Public Methods
Protected Methods
Private Methods